internal/runtimehandlerhooks/high_performance_hooks.go

```diff
@@ -92,6 +96,36 @@ func (h *HighPerformanceHooks) PreStart(ctx context.Context, c *oci.Container, s
 		}
 	}
 
+	// configure c-states for the container CPUs
+	configureCState, value := shouldCStatesBeConfigured(s.Annotations())
+	if configureCState {
```

Comment:
nit: perhaps these two can be combined (added benefit is limiting the variables scope:

```go
if ok, value := shouldCStatesBeConfigured(s.Annotations()); ok {
...
}
```

Note that I call the variable `ok` as it is only used in this one line, and it's clear it is related to `shouldCStatesBeConfigured`. In general, the shorter the scope, the shorter the variable name should be.

Reply:
OK - I made the change - but I named the variable "configure" instead of "ok" as I personally read "ok" as a pass/fail thing where this return as more meaning.